hibernate-core/src/main/java/org/hibernate/boot/model/source/spi/CollectionIdSource.java

```diff
@@ -30,4 +32,9 @@
 	 * @return The identifier value generator name
 	 */
 	public String getGeneratorName();
+
```

Comment:
Could the addition of this method to CollectionIdSource interface break an application (e.g., tools) in 5.0 or 5.1?